pontoon/api/README.md

```diff
@@ -1,13 +1,16 @@
 # GraphQL API
 
 Pontoon exposes some of its data via a public API endpoint. The API is
-[GraphQL](http://graphql.org/)-based and available at `/graphql/`.
+[GraphQL](https://graphql.org/)-based and available at `/graphql/`.
 
-## Production Deployments
+The endpoint has two modes of operation: a JSON one and an HTML one.
 
-When run in production (`DEV is False`) the API returns `application/json`
-responses to GET and POST requests. In case of GET requests, any whitespace in
-the query must be escaped.
+## JSON mode
+
+When a request is sent without any headers, with `Accept: application/json` or
+if it explicitly contains a `raw` query argument, the endpoint will return JSON
+`application/json` responses to GET and POST requests. In case of GET requests,
+any whitespace in the query must be escaped.
```

Comment:
@MikkCZ Could you plese verify if these changes to the GraphQL endpoint interfere with the Pontoon Add-On?

Reply:
This PR is deployed to stage:
https://mozilla-pontoon-staging.herokuapp.com/graphql

Reply:
Hi. Thanks for the ping. I pulled the GraphQL schema, generated types into Pontoon Add-on, and all seems good.

And if the GraphQL schema now will be made available by the production instance as well, I will be able to simplify the tooling for Pontoon Add-on as well.

Reply:
Thanks for checking!

We should revisit our API strategy. You aren't coming to FOSDEM this year by any chance? :)

Reply:
No, at the time of FOSDEM I will be closer to Africa.